embedly/gunicorn.conf

```diff
@@ -1,5 +1,6 @@
 import multiprocessing
 
+access_logfile = "-"
```

Comment:
but understandably, having this as stdout here may make troubleshooting possible in stage